WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift

```diff
@@ -885,8 +859,9 @@ private extension StorePickerViewController {
                                                             comment: "Button to connect to an existing store from the store picker")
         static let cancel = NSLocalizedString("Cancel",
                                               comment: "Button to dismiss the action sheet on the store picker")
-        static let addStoreButton = NSLocalizedString("Add a Store",
```

Comment:
We should also remove the `"Create a new store" string.

Reply:
This is handled in https://github.com/woocommerce/woocommerce-ios/pull/12376/commits/d114f8bc024b6f314305f5d9b54d20bced508bca